lexos/templates/scrub.html

```diff
@@ -34,6 +34,7 @@
                 }
                                 });
            tagDict=JSON.stringify(tagDict);
+           console.log(tagDict);
```

Comment:
I will investigate browser debuggers. `editorconfig` is already installed in vs code, but I just added it to Sublime.